tests/import_source/import_source.cpp

```diff
@@ -177,38 +177,38 @@ TEST(ImportSource, importSourceMove)
 
 TEST(ImportSource, importSourceSharedAcrossModels)
 {
-    auto m1 = libcellml::Model::create("m1");
+    auto i1 = libcellml::Importer::create();
     auto imp = libcellml::ImportSource::create();
 
-    EXPECT_TRUE(m1->addImportSource(imp));
+    EXPECT_TRUE(i1->addImportSource(imp));
 
     // Add import source to another model.
```

Comment:
Add import source to another IMPORTER. and similar comments throughout...

Reply:
2dec745